package.json

```diff
@@ -1090,9 +1091,11 @@
     "moment": "^2.19.3",
     "ms-rest": "^2.3.8",
     "ms-rest-azure": "^2.5.9",
+    "npm": "^6.8.0",
```

Comment:
What's this for?

Reply:
And build and run?

Reply:
These must have come up when I ran `npm run all`. I'll remove these.

Reply:
Not present on the new PR.